Intersect (Core)/Config/Options.cs

```diff
@@ -56,6 +56,8 @@ public class Options
 
         [JsonProperty("Loot")] public LootOptions LootOpts = new LootOptions();
 
+        [JsonProperty("Sprites")] public SpriteOptions SpriteOpts = new SpriteOptions();
```

Comment:
There is no `Sprites` field to conflict with this, just call it `Sprites` and drop the `JsonProperty` attribute since it won't be necessary.

Reply:
Just as an aside, shouldn't changing this for the other options categories become a cleanup issue then?

Reply:
Maybe? We should decide on this. Changing the name of this variable only would make referencing options pretty inconsistent.

Reply:
I plan on cleaning up the configuration rather soon anyway, I'm just trying to not add more code that we're just going to redo later on.

Reply:
Done.